src/Nethermind/Nethermind.Evm/Instruction.cs

```diff
@@ -185,12 +188,12 @@ public enum Instruction : byte
     
     public static class InstructionExtensions
     {
-        public static string? GetName(this Instruction instruction, bool isPostMerge = false)
-        {
-            return (instruction == Instruction.PREVRANDAO && !isPostMerge)
+        public static string? GetName(this Instruction instruction, bool isPostMerge = false) =>
+            (instruction == Instruction.PREVRANDAO && !isPostMerge)
                 ? "DIFFICULTY"
-                : System.Enum.GetName(typeof(Instruction), instruction); //TODO: This is slow! Generate switch?
-        }
+                : FastEnum.IsDefined(instruction)
+                    ? FastEnum.GetName(instruction)
+                    : null;
```

Comment:
Because of: https://github.com/xin9le/FastEnum/issues/11  , might get fix at next release